src/main/java/ch/njol/skript/expressions/ExprAmount.java

```diff
@@ -41,7 +41,7 @@
  */
 @Name("Amount")
 @Description({"The amount of something.",
-		"Please note that <code>amount of &lt;items&gt;</code> will not return the number of items, but the number of stacks, e.g. 1 for a stack of 64 torches."})
+		"Please note that <code>amount of %items%</code> will not return the number of items, but the number of stacks, e.g. 1 for a stack of 64 torches."})
```

Comment:
I'd recommend adding a reference to the `item amount` expression here:
```suggestion
		"Please note that <code>amount of %items%</code> will not return the number of items, but the number of stacks, e.g. 1 for a stack of 64 torches. To get the amount of items in a stack, see the <a href=expressions#ExprItemAmount>item amount expressions</a>."})
```

Reply:
@Efnilite you know you can just click accept suggestion right lol

Reply:
Yeah but then I have to make like 90 commits, just think it's easier :)

Reply:
Not that it matters, we squash the PRs to a single commit so.